setup.py

```diff
@@ -1,22 +1,45 @@
-from os import path
+import os
+import subprocess
+import sys
 
 from setuptools import setup
+from setuptools.command.install import install
 
-FULLVERSION = '0.0.1'
+FULLVERSION = '0.0.2-2'
 VERSION = FULLVERSION
 
+class PostInstall(install):
```

Comment:
can you add a short comment on what this does (for posterity)? :D

Reply:
Added in a7734de